lib/template.html

```diff
@@ -65,6 +65,10 @@
   {% assign seo_author_twitter = seo_author_twitter | replace:"@","" %}
 {% endif %}
 
+{% if page.date_modified or page.last_modified_at or page.date %}
+  {% assign seo_date_modified = page.seo.date_modified | default: page.last_modified_at | default: page.date %}
```

Comment:
The difference between this plugin and The Verge is that they control every aspect of their site, and this plugin has to serve a more general use case. We have no way of knowing that a document’s date is the same as (or even a good approximation of) its updated date. I assert that the absence of data is better than made up daya.

When you say “Google will complain,” do you mean validation fails, or Google complains loudly? If validation fails, I guess we'll do what's needed to fix that. If it's merely a warning, that seems like A Good Thing! Publishers _should_ be made aware that including modified information is an important thing to do, and we can't just hand-wave over that.